ang/crmMosaico/TemplateItem.js

```diff
@@ -12,13 +13,26 @@
       },
       link: function($scope, $el, $attr) {
         var ts = $scope.ts = CRM.ts('mosaico');
-        $scope.$watch('crmMosaicoTemplateItem', function(newValue){
+
+        $scope.$watch('crmMosaicoTemplateItem', function(newValue) {
           $scope.myOptions = newValue;
+
+          // Template default action label based on current state
+          if (newValue['state']) {
```

Comment:
Fixed in 
https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/5/commits/99b27c334a1d8566afe9852593ff77f94080c3fe